src/exoticatechnologies/util/datastructures/ExoticStuffHolder.kt

```diff
@@ -0,0 +1,15 @@
+package exoticatechnologies.util.datastructures
+
+import com.fs.starfarer.api.fleet.FleetMemberAPI
+import exoticatechnologies.modifications.ShipModifications
+import exoticatechnologies.modifications.exotics.Exotic
+import exoticatechnologies.modifications.exotics.ExoticData
+
+/**
+ * Utility class for holding the necessary baggage every [Exotic] needs, namely the [member], [mods] and [exoticData]
+ */
+data class ExoticStuffHolder(
```

Comment:
delete this. we don't need it anymore, and can always put it back in if/when there's an actual need for it

Reply:
deleted. resolving.